src/rust/ide/view/graph-editor/src/component/visualization/container.rs

```diff
@@ -602,6 +604,11 @@ impl Container {
             eval_ on_selected ( action_bar.hide_icons.emit(()) );
         }
 
+        // FIXME[mm]: If we set the size right here, we will see spurious shapes in some
+        // computation heavy circumstances (e.g., collapsing nodes, or creating an new project).
+        // If we let the animation do the size change, the size will only be set after the delays.
+        // This is not optimal the optimal solution to this problem, as it also means that we have
+        // an animation on an invisible component running.
```

Comment:
This is from another PR I believe? And fix `This is not optimal the optimal solution`

Reply:
Sorry, yes there was a commit from a different PR here. That is gone now.